packages/blockchain-extension/extension/util/ExtensionsInteractionUtil.ts

```diff
@@ -19,14 +19,19 @@ import { CloudAccountApi } from '../interfaces/cloud-account-api';
 import { VSCodeBlockchainOutputAdapter } from '../logging/VSCodeBlockchainOutputAdapter';
 import { LogType } from 'ibm-blockchain-platform-common';
 import { URL } from 'url';
+import { DependencyProperties } from '../dependencies/Dependencies';
 
 /*
  * This file will hold functions to allow interaction with other extensions.
  */
 export class ExtensionsInteractionUtil {
 
+    // public static isIBMCloudExtensionInstalled(): boolean {
+    //     return !!vscode.extensions.getExtension(DependencyProperties.IBM_CLOUD_ACCOUNT_EXTENSION);
+    // }
+
```

Comment:
I saw you also left this code commented out in the original v2 PR. Is there a reason for this?

Reply:
Yes it is there for when the IBM Cloud Account functionality can be added back in (we're waiting for an update to something)